test/evergreen.yml

```diff
@@ -13,7 +13,7 @@ functions:
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${build_id}.tgz
+        remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${parent_task|compile}_${build_id}.tgz
```

Comment:
The term `parent_task` sounds a bit confusing to me. Could we use something else e.g. `dependent_task`?

Reply:
dependent task sounds it can be multiple but really we can only specify one.